default/stringtables/en.txt

```diff
@@ -995,6 +995,8 @@ Too many human players.
 ERROR_CONNECTION_WAS_REPLACED
 Your connection was replaced.
 
+ERROR_CANNOT_ELIMINATE_NO_HUMAN_PLAYER
+Cannot eliminate other than Player
```

Comment:
This error message is confusing.  Change to something like.
Only human clients can concede.

Reply:
Actually, we were just talking about letting the AI's concede this way also when they've lost their last colony (as an interim measure while we explore more interesting ways for them to surrender).  If the point of the section of code that uses this string was to prevent AI's from conceding, then please just delete that whole section of code.

I suspect, though that the point was more to prevent an Observer or Moderator from accidentally hitting the concede button (which should probably be disabled for them anyways),

I'd suggest changing the label to "ERROR_NONPLAYER_CANNOT_CONCEDE", change the message to "Only Players can concede" and change the referencing code to accept the concession from both human clients and AI clients.

Reply:
Changed label and allowed an AI to eliminate itself.